en/docs/develop/components/rest-api.md

```diff
@@ -36,56 +36,44 @@ Choreo allows you to design your REST API in three possible ways as shown in the
-
-### Low-code mode
-
-The following image shows the low-code diagram view in Choreo for a simple API designed to retrieve information such as population and GDP for a specified country. Here, the [World Bank connector](https://lib.ballerina.io/ballerinax/worldbank/1.2.0) extracts the required data and sends the extracted data as a JSON response.
-
-![Low-code view](../../assets/img/rest-apis/low-code-view.png){.cInlineImage-full}
-
-The low-code programming model allows developers to use general programming constructs such as loops, conditional statements, variable declarations and assignments, logs, data transformations(visual data mapping), and more in a graphical editor. It also allows developers to connect to built-in or custom-developed connectors.
+Choreo supports spec-driven API development. Once you have carefully designed and defined your API, you can start implementing the functionality of your API. Choreo allows you to link a GitHub project when creating the REST API. You can implement your API as a [Ballerina](https://ballerina.io/) project or as in any language as a Docker container and store in the GitHub repo. Alternatively, if you are selecting the Ballerina preset when creating the REST API, you can link an empty GitHub repository, and then proceed to implement it before deploying the REST API. 
 
-The main advantage of the low-code programming mode is that it can increase developer productivity multiple folds by eliminating the need to write complex code. As developers use constructs on the low-code editor to implement the API, the IDE automatically writes the corresponding [Ballerina](https://ballerina.io/) source code for the API, which significantly speeds up the developer learning process and boosts overall productivity.
+WSO2 provides a [Ballerina extension for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=WSO2.ballerina) which you can use to eaasily develop your REST API as a Ballerina project.
 
-### Pro-code mode
-
-The pro-code mode of Choreo allows developers to directly edit code in a manner where the graphical representation in the low-code editor gets updated accordingly. Choreo also allows viewing the low-code and pro-code editors side-by-side.
+## Lifecycle of a REST API
 
-![Low code and pro-code view](../../assets/img/rest-apis/low-code-and-pro-code-view.png){.cInlineImage-full}
+Once you implement a REST API, it needs to be built, tested, and eventually put into production. You can apply any of the six API lifecycles states (i.e., created, pre-released, published, blocked, deprecated, and retired) while adhering to their respective lifecycle flows.
```

Comment:
Suggestion:
"Once you implement a REST API, you should build it, test it, and eventually promote it to the production environment. You can apply any of the six API lifecycle states (i.e., Created, Pre-released, Published, Blocked, Deprecated, and Retired) while adhering to their respective lifecycle flows."

Addressing the following with the suggestion:
- Reducing the use of passive voice.
- Since we are specifically identifying that list of words as states, we should have them starting with caps.